### DISCUSSION TITLE
EarTrumpet unable to change playback device for System Sounds?

### DISCUSSION THREAD

**Discussion by samwzlim** [File-New-Project/EarTrumpet#795] [Q&A]
I would like to route System Sounds to my monitor's speaker, as I have dedicated external speakers that I wish to use for playing music only. Is this possible? At the moment, it seems like System Sounds is set to the default playback device, which in this case, are my external speakers. I could set my monitor's speaker as the default playback device, but I would have to manually switch all other app audio output to my external speakers, which would be very tedious.

**Comment by davuxcom** [comment]
Windows doesn't allow the system sounds session to be moved to another device.
